cmd/conformance-tester/pkg/runner/waiters.go

```diff
@@ -122,7 +122,7 @@ func waitUntilAllPodsAreReady(ctx context.Context, log *zap.SugaredLogger, opts
 		unready := sets.New[string]()
 		for _, pod := range podList.Items {
 			// Ignore pods failing kubelet admission (KKP #6185)
-			if !util.PodIsReady(&pod) && !podFailedKubeletAdmissionDueToNodeAffinityPredicate(&pod, log) {
+			if !util.PodIsReady(&pod) && !podFailedKubeletAdmissionDueToNodeAffinityPredicate(&pod, log) && !util.PodIsCompleted(&pod) {
```

Comment:
here the tests expected all user cluster pods to be in a state `Ready`, however, hubble adds a pod that ultimately succeeds and goes to `Completed` state. Added a small commit https://github.com/kubermatic/kubermatic/pull/12752/commits/33e8f4854fd25d5aa6cb5cd692992e26449a27ad permitting that.
```
{  context deadline exceeded; last error was: not all Pods are ready: [hubble-generate-certs-lcb7z]}
```
```
kube-system            hubble-generate-certs-l6x9h                  0/1     Completed   0             10m
```